RInAppMessaging/Classes/Views/FullScreenView.swift

```diff
@@ -32,7 +32,11 @@ internal class FullScreenView: FullView {
 
         let statusBarBackground = UIView()
         statusBarBackground.translatesAutoresizingMaskIntoConstraints = false
-        statusBarBackground.backgroundColor = UIColor.white.withAlphaComponent(0.4)
+        if UIApplication.shared.getCurrentStatusBarStyle() == .lightContent {
+            statusBarBackground.backgroundColor = UIColor.black.withAlphaComponent(0.4)
+        } else {
+            statusBarBackground.backgroundColor = UIColor.white.withAlphaComponent(0.4)
+        }
```

Comment:
You could have something like:
statusBarBackground.backgroundColor = .statusBarColor

where statusBarColor is a computed property:
statusBarColor = {
         if UIApplication.shared.getCurrentStatusBarStyle() == .lightContent {
             return UIColor.black.withAlphaComponent(0.4)
         } else {
             return UIColor.white.withAlphaComponent(0.4)
         }
}